article/app/controllers/LiveBlogController.scala

```diff
@@ -13,14 +12,14 @@ import model.dotcomrendering.{DotcomRenderingDataModel, PageType}
 import model.liveblog.BodyBlock
 import model.liveblog.BodyBlock.{KeyEvent, SummaryEvent}
 import model.{ApplicationContext, CanonicalLiveBlog, _}
-import org.joda.time.{DateTime, DateTimeZone}
 import pages.{ArticleEmailHtmlPage, LiveBlogHtmlPage, MinuteHtmlPage}
 import play.api.libs.ws.WSClient
 import play.api.mvc._
 import play.twirl.api.Html
 import renderers.DotcomRenderingService
 import services.CAPILookup
 import services.dotcomponents.DotcomponentsLogger
+import topmentions.{TopMentionsService}
```

Comment:
Maybe outside of the scope of tihs PR but the TopMentionsService should probably go in the `services` package!

Reply:
Sounds a good idea. Do you think maybe the `TopMentionsS3Client` can also be moved to the services?

Reply:
Hm, not sure! I don't mind either way, there seems to be some precedent to put clients into the services package too.

Reply:
Sure, If that's OK, I'll move them to the services in the upcoming PR just to reduce the conflicts with another parallel PR Anna is working on